index.html

```diff
@@ -0,0 +1,44 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="main.css">
+    <title>Portfolio-Mobile</title>
+</head>
+
+<body>
+    <header>
+        <a href="#" class="name">Victor</a>
+        <nav>
+            <img src="pictures/Enabled.png" alt="menu">
```

Comment:
- [ ] Instead of using pictures you can use icons from [font-awesome](https://fontawesome.com/start). This is optional but ı highly recommended. It is not hard to using.